.github/workflows/pr-code-format.yml

```diff
@@ -34,7 +34,6 @@ jobs:
         uses: actions/checkout@v4
         with:
           reository: ${{ github.repository }}
-          ref: ${{ github.base_ref }}
```

Comment:
This was probably here for testing? Needs to be removed before pushing.

Reply:
Yeah, I'll drop that commit.